### ISSUE TITLE
Basepackages Wizard

### ISSUE THREAD

**Issue by oyeaussie** [sp-framework/core#280]
Right now wizard is being handled by JS and AdminLTE template. 

Wizards are necessary in today's app development. A guided steps to achieve something.

An example would be Checkout process. This is a basic requirement of all ecom apps. 

The walk through of the checkout process can be handled by the backend and not by the Ajax. This way, we can keep a track of where the process is and what needs to be done at that time. The frontend can only do 2 things, give data to backend and take data and/or processed html content back from the server.

Basepackages process can be utilized. Wizard can have predefined methods that can be used by the front end developers and those methods can be initialized by the front end devs. On the bases of chosen methods, the wizard basepackage can add those methods into process and keep a track of them them there. Process file can include sessionID, userID, etc to keep a track of where the process is, so even if the connection breaks, user can resume where they left.

######## Issue imported from Gitea ########
# Details
Gitea Issue ID : 338
State : open
Created : 2023-04-07T14:40:20+10:00
# Issue Description
Right now wizard is being handled by JS and AdminLTE template. 

Wizards are necessary in today's app development. A guided steps to achieve something.

An example would be Checkout process. This is a basic requirement of all ecom apps. 

The walk through of the checkout process can be handled by the backend and not by the Ajax. This way, we can keep a track of where the process is and what needs to be done at that time. The frontend can only do 2 things, give data to backend and take data and/or processed html content back from the server.

Basepackages process can be utilized. Wizard can have predefined methods that can be used by the front end developers and those methods can be initialized by the front end devs. On the bases of chosen methods, the wizard basepackage can add those methods into process and keep a track of them them there. Process file can include sessionID, userID, etc to keep a track of where the process is, so even if the connection breaks, user can resume where they left.

# Timeline
Label : Added Proposal on 2023-04-07T14:40:20+10:00.
Label : Added P4 on 2023-04-07T14:40:20+10:00.
